Procfile

```diff
@@ -0,0 +1,2 @@
+worker: celery worker -A doorman.worker:celery -l INFO
+web: python manage.py runserver -h '0.0.0.0' -p $PORT
```

Comment:
`DOORMAN_ENV` variable is not used, instead of that all Heroku-hosted instances (which are determined by presence of `DYNO` variable) are considered production (because they are publically available at `<app-name>.herokuapp.com`) and use separate configuration class.
`DOORMAN_SETTINGS` variable is not used as well; instead all configuration is done through environment variables, which is the default way for Heroku. Please see [HerokuConfig class](https://github.com/mwielgoszewski/doorman/pull/70/files#diff-dc715b0545c788442fa78cc8dc8b8b15R263).